testutils/src/main/scala/za/co/absa/enceladus/testutils/e2eSparkRunner/E2ESparkRunner.scala

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.testutils.e2eSparkRunner
+
+import org.apache.log4j.{LogManager, Logger}
+import za.co.absa.enceladus.testutils.HelperFunctions
+
+import sys.process._
+import scala.io.Source
+
+object E2ESparkRunner {
+  private val log: Logger = LogManager.getLogger(this.getClass)
+
+  private final val stdJarPath: String = "$DCE_JAR_PATH/$DCE_STD_SPT_JAR"
```

Comment:
Tiny: I think a `private val` in Scala is pointless to also mark as `final` (multiple occurrences)

Reply:
removed